<launch>

    <group ns="stereo_vision" clear_params="true">
        <arg name="camera_config" value="OV5647_MIPI_2Lane_RAW8_8b_2592x1944_AB.cfg" /> <!-- Config file in camera_config_files folder of your camera. -->
        <arg name="serial_number" value="" /> <!-- Serial number of the camera in the form of "XXXX-XXXX-XXXX", leave it blank "" if there is only one camera. -->
        <arg name="horizontal_flip" value="true" /> <!-- Flip image horizontally. -->
        <arg name="vertical_flip" value="false" /> <!-- Flip image vertically. -->
        <arg name="frame_id" value="stereo_vision" /> <!--Topic frame_id. -->

        <node name="stereo_vision_arducam_node" pkg="stereo_vision_ros" type="arducam_ros_node.py" output="screen">
            <param name="config_file" value="$(find stereo_vision_ros)/camera_config_files/$(arg camera_config)" />
            <param name="camera_serial" value="$(arg serial_number)" />
            <param name="horizontal_flip" value="$(arg horizontal_flip)" />
            <param name="vertical_flip" value="$(arg vertical_flip)" />
            <param name="frame_id" value="$(arg frame_id)" />
        </node>
    </group>

</launch>
